frontend/src/pages/modelServing/customServingRuntimes/CustomServingRuntimeTableRow.tsx

```diff
@@ -46,7 +46,7 @@ const CustomServingRuntimeTableRow: React.FC<CustomServingRuntimeTableRowProps>
         <ResourceNameTooltip resource={template}>
           {getServingRuntimeDisplayNameFromTemplate(template)}
         </ResourceNameTooltip>
-        {templateOOTB && <Label>Pre-installed</Label>}
+        {templateOwnedByDSC && <Label data-testid="pre-installed-label">Pre-installed</Label>}
```

Comment:
Maybe a little out of scope but you could use the `PreInstalledName` const from https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/concepts/k8s/utils.ts#L5